### PR TITLE
Noissue - Add config docs

### PR DESCRIPTION
**Pull request by WashingtonKK**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Adds documentation for CLI Config command

### Which issue(s) does this PR fix/relate to?
NOISSUE

### List any changes that modify/break current functionality
N/A

### Notes
To be merged after https://github.com/mainflux/mainflux/pull/1858

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -607,4 +608,102 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
...
+mainflux-cli groups get all <user_token>
+```
+
+#### Get Group Members
+
+```bash
+mainflux-cli groups members <group_id> <user_token>
+```
+
+#### Get Memberships
```
Comment (SammyOina) [inline docs/cli.md]: are the changes for groups missing or meant to be in this PR

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Comment by SammyOina** [comment]
use rebase rather than merge
image

`git config pull.rebase true`

File: docs/cli.md [hunk 2]
```diff
@@ -607,4 +608,102 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
...
+
+```bash
+mainflux-cli groups membership <member_id> <user_token>
+```
+
+#### Assign Members to Group
+
+```bash
+mainflux-cli groups assign <member_ids> <member_type> <group_id> <user_token>
```
Comment (ianmuchyri) [inline docs/cli.md]: check on the new updates on` mainflux-cli assign`

File: docs/cli.md (hunk 1 above)
Comment (WashingtonKK) [reply docs/cli.md]: I have removed them, it was an error.

File: docs/cli.md (hunk 2 above)
Comment (WashingtonKK) [reply docs/cli.md]: Okay

Comment (WashingtonKK) [reply docs/cli.md]: Is that supposed to be in this PR?

File: docs/cli.md [hunk 3]
```diff
@@ -607,4 +608,102 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
...
+
+```bash
+mainflux-cli groups get <group_id> <user_token>
+```
+
+#### Get Groups
+
+```bash
+mainflux-cli groups get all <user_token>
+```
```
Comment (ianmuchyri) [inline docs/cli.md]: add the other `get` commands: `children` and `parents`

File: docs/cli.md (hunk 2 above)
Comment (WashingtonKK) [reply docs/cli.md]: @ianmuchyri

File: docs/cli.md (hunk 3 above)
Comment (ianmuchyri) [reply docs/cli.md]: also, add the` update` command to update a group's details

Comment (WashingtonKK) [reply docs/cli.md]: Is this supposed to be in this PR? @ianmuchyri, or this one is just to add cli config  docs

Comment (ianmuchyri) [reply docs/cli.md]: Oh, yeah, you can create another pr on that

Comment (ianmuchyri) [reply docs/cli.md]: I hadn't seen you had removed them

File: docs/cli.md (hunk 2 above)
Comment (ianmuchyri) [reply docs/cli.md]: On another. You can resolve this comment

File: docs/cli.md (hunk 3 above)
Comment (ianmuchyri) [reply docs/cli.md]: On another. You can resolve this comment

File: docs/cli.md [hunk 4]
```diff
@@ -607,4 +607,46 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
...
+mainflux-cli config <parameter> <value>
+```
+
+Response should look like this:
+
+```bash
+  ok 
+```
+
+This command is used to set the flags to be used by CLI in a local TOML file. The default location of the TOML file is near the binaries. To change the location of the TOML file you can run the command:
```
Comment (dborovcanin) [inline docs/cli.md]: `The default location of the TOML file is near the binaries` -> `is in the same directory as the CLI binary`

File: docs/cli.md [hunk 5]
```diff
@@ -607,4 +607,46 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
 mainflux-cli bootstrap bootstrap <external_id> <external_key> -b <bootstrap-url>
 ```
 
-[releases]: https://github.com/mainflux/mainflux/releases
+## Config
```
Comment (dborovcanin) [inline docs/cli.md:610]: Add some explanation here, like:
Mainflux CLI tool supports configuration files that contain some of the basic settings so you don't have to specify them through flags.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — DISMISSED** [review]

**Review by SammyOina — APPROVED** [review]

File: docs/cli.md [hunk 6]
```diff
@@ -607,4 +607,47 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
...
+
+```
+  mainflux-cli config <parameter> <value> -c "cli/file_name.toml"
+```
+
+
+The possible parameters that can be set using the config command are:
+
+```
+  - bootstrap_url
```
Comment (drasko) [inline docs/cli.md]: Kill this space between `-` and the option

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/cli.md [hunk 7]
```diff
@@ -607,4 +607,47 @@ mainflux-cli bootstrap remove <thing_id> <user_token> -b <bootstrap-url>
...
+
+```
+  mainflux-cli config <parameter> <value> -c "cli/file_name.toml"
+```
+
+
+The possible parameters that can be set using the config command are:
+
+```
+  -bootstrap_url
```
Comment (drasko) [inline docs/cli.md]: Kill empty space behind the string. Add description like for any Linux tool.

Comment (drasko) [reply docs/cli.md]: Add a table with description

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/cli.md (hunk 7 above)
Comment (WashingtonKK) [reply docs/cli.md]: Okay

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**